subsys/bluetooth/common/Kconfig

```diff
@@ -183,12 +183,40 @@ config BT_BUF_CMD_TX_SIZE
 	  to the maximum of 255.
 
 config BT_BUF_CMD_TX_COUNT
-	int "Number of HCI command buffers"
-	default 2
-	range 2 64
+	int "Number of HCI command buffers" if !HAS_BT_CTLR
+	# !BT_HCI_ACL_FLOW_CONTROL ==> Permit configurable Num_HCI_Command_Packets for Host only
+	# build but hidden when HAS_BT_CTLR (default 1 for Controller only build).
```

Comment:
```suggestion
	# build but hidden when HAS_BT_CTLR=n (default 1 for Controller only build).
```

Reply:
Hidden when `HAS_BT_CTLR=y`, then the `BT_BUF_CMD_TX_COUNT` be calculated.

Reply:
Right, my bad :D